src/main/java/at/hannibal2/skyhanni/api/enoughupdates/EnoughUpdatesManager.kt

```diff
@@ -185,6 +185,7 @@ object EnoughUpdatesManager {
             return fromJson
         } catch (e: Exception) {
             ErrorManager.logErrorWithData(e, "Error converting nbt to json", "malformed nbt" to convertedNbt, "original nbt" to nbtString)
+            println("Error converting nbt to json: ${e.message}\nMalformed NBT: $convertedNbt\nOriginal NBT: $nbtString")
```

Comment:
wrong branch?

Reply:
not really, it was in this branch that it happened, i figured i'd leave it - since the `logErrorWithData()` seemed to do jack shit in terms of providing useful info;

<img width="786" height="171" alt="image" src="https://github.com/user-attachments/assets/62be85a1-33e5-4b3b-ae40-cf4d2cc3e80a" />
<img width="991" height="182" alt="image" src="https://github.com/user-attachments/assets/e5bd3dd2-f9a6-43ea-a380-218bccd382b7" />

https://discord.com/channels/997079228510117908/1117409804382642296/1402797363889967276